test-apps/nextjs-app-apple/README.md

```diff
@@ -0,0 +1,40 @@
+# Lucia + Next.js demo
+
```

Comment:
Can you remove the entire username/pasword auth from the project? Seems better to just focus on the Apple ID part.

Reply:
i think this is important to show various types of auth working together

Reply:
It’s just more to maintain, and the other examples should be enough if you want to see how they work together.